wasm_src/ast_wrapper/post.ts

```diff
@@ -194,6 +196,54 @@ Module.transformPointArrays = function (wcsInfo: number, xIn: Float64Array, yIn:
     return result;
 };
 
+Module.getGeodesicPointArray = function (wcsInfo: number, npoint: number, start: {x: number, y: number}, finish: {x: number, y: number}) {
+    // Return empty array if arguments are invalid
+    const yIn = new Float64Array([start.y, finish.y]);
+    const xIn = new Float64Array([start.x, finish.x]);
+
+    if (!(xIn instanceof Float64Array) || !(yIn instanceof Float64Array) || xIn.length !== yIn.length) {
+        return {x: new Float64Array(1), y: new Float64Array(1)};
+    }
+
+    // Allocate and assign WASM memory
+    const N = xIn.length;
+    const xInPtr = Module._malloc(N * 8);
+    const yInPtr = Module._malloc(N * 8);
+    const outPtr = Module._malloc(201 * 2 * 8);
```

Comment:
Could we use `npoint` for `201`? Same for all `201` in `getGeodesicPointArray` and `getAxisPointArray`.